src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -29,7 +29,10 @@ public TaskDto getTask(@RequestParam Long taskId) throws TaskNotFoundException {
     }
 
     @RequestMapping(method = RequestMethod.DELETE, value = "deleteTask")
-    public void deleteTask(Long taskId){
+    public void deleteTask(@RequestParam Long taskId){
+        System.out.println("Delete: " + "id: "+getTask(taskId).getId() + " title: " + getTask(taskId).getTitle()
+                + " content: " + getTask(taskId).getContent());
```

Comment:
to dalej jest jest poprawione. Sa tutaj 2 problemy:
- test nie jest idempotentny
- 3 x wywolana jests metoda getTask i 3x idzie zapytanie do bazy danych